android/app/src/main/java/app/organicmaps/editor/EditorFragment.java

```diff
@@ -540,8 +542,6 @@ else if (id == R.id.more_names || id == R.id.show_additional_names)
     }
     else if (id == R.id.add_langs)
       mParent.addLanguage();
-    else if (id == R.id.about_osm)
-      Utils.openUrl(requireActivity(), getString(R.string.osm_wiki_about_url));
```

Comment:
Is the text/url clickable now?

Reply:
Yes, the blue text is clickable. The removed code is form the old separate 'About OpenStreetMap' textview.